book/website/collaboration/team-manual.md

```diff
@@ -0,0 +1,203 @@
+(cl-team-manual)=
+# Team Manuals
+
+Team manuals describe expectations for team members, highlight resources and provides documentation on processes. 
+
+(cl-team-manual-benefits)=
+## Benefits of Team Manuals
+
+- Clarity about expectations
+- Everyone has access to the same information
+- Everyone can contribute to the team manual and updating will take less time
+- Discussions around ways of working are normalised
+- Group members will think more actively about how to do research
+- Answers to frequently asked questions will be available (knowledge management)
+- Can contribute to a safer lab environment (technically and socially)
+- You can hold people more easily accountable (see below)
+
+(cl-team-manual-content)=
+## What to include in a Team Manual? 
+
+### Main points 
+1. Mission/vision statement:
+What are the research goals of the team, why are these the goals? 
+How does the team work? What is the history of the group? 
+How does the team fit into the department/institute? 
+What are the [principles](https://www.lesleahlusko.org/principles-of-community-for-our-intellectual-family/)? 
+How do we do science in a way that fits with our mission/vision statement. 
+2. Roles and expectations: What are the responsbilities of each team member, including the PI? 
+When will these roles be discussed when a new member joins the team?
+
+### More details
+- How is the lab manual updated? 
+Who can make propose the changes (ideally every member) and what is the process? 
+Do you keep multiple versions of the lab manual? 
+Where will the lab manual be made available for all lab members to access?
+- Open Science: 
+How does the team manage and share code, data, materials? 
+How is documentation done? 
+- Lab manual: 
```

Comment:
Each bullet point has an introductory term such as "Communication", "Behaviour", etc - this introductory term is "Lab manual". Given that the entire chapter is on "Team Manuals", and that the content of this bullet point is on internal research processes, maybe we can rename this introductory term to something like this?

```suggestion
- Research process: 
```